src/simulation/monsters/special/Barrows.ts

```diff
@@ -82,4 +82,9 @@ export class Barrows extends Monster {
 }
 
 // Uses NPC id for Dharoks
-export default new Barrows({ id: 1673, name: 'Barrows', aliases: ['barrows'] });
+export default new Barrows({
+	id: 1673,
+	name: 'Barrows',
+	table: new LootTable().add(BarrowsTable).add(OtherTable),
```

Comment:
I dont understand the point of this file change, because this `table` is completely incorrect, and the `kill` method is still there?

Reply:
This is meant to fix +cl barrows completely failing. Currently, it returns an error as it's trying to access the table which is not defined.
For the kill method, since you can't get duplicate barrows pieces in a chest, I believe changing the kill method would be a lot more difficult.

Reply:
oh right, that makes sense, although it still feels wrong to me because now `table` is incorrect to what the table actually is. however, `+cl barrows` now works anyway as it was just made as its own cl page, rather than the bot trying to dynamically look up its table as a monster, so the change is not needed anymore anyway. The vorkath one looks totally like a great change though.

Reply:
I've reverted the changes to the barrows tables.